src/server/scripts/EasternKingdoms/OperationGnomeregan/operation_gnomeregan.cpp

```diff
@@ -0,0 +1,2565 @@
+                        AddWaypoint(6, -5039.229492f, 739.272888f, 256.475586f);
+                        AddWaypoint(7, -4948.179199f, 731.976929f, 260.396484f);
+                        AddWaypoint(8, -4946.559570f, 731.800903f, 261.645752f);
+                        AddWaypoint(9, -4943.546875f, 731.456848f, 261.645752f);
+                        AddWaypoint(10, -4944.289063f, 735.979248f, 261.645752f);
+                        AddWaypoint(11, -4945.196289f, 733.968018f, 261.645752f);
+                        break;
+                    case 5:
+                        AddWaypoint(0, -5085.312500f, 477.487366f, 401.958099f);
+                        AddWaypoint(1, -5093.171387f, 469.209137f, 403.715790f);
+                        AddWaypoint(2, -5100.262695f, 462.922180f, 403.585724f);
+                        AddWaypoint(3, -5106.192871f, 458.708801f, 402.284393f);
+                        AddWaypoint(4, -5113.296875f, 454.575165f, 400.071075f);
+                        AddWaypoint(5, -5129.138184f, 448.644257f, 395.268951f);
+                        AddWaypoint(6, -5144.433594f, 458.945282f, 392.658112f);
+                        AddWaypoint(7, -5157.008301f, 472.717865f, 390.544586f);
+                        AddWaypoint(8, -5167.264160f, 479.733826f, 389.530670f);
+                        AddWaypoint(9, -5182.246582f, 490.885254f, 388.022522f);
+                        AddWaypoint(10, -5188.400391f, 508.496002f, 387.779266f);
+                        AddWaypoint(11, -5189.048828f, 533.551758f, 389.216064f);
+                        AddWaypoint(12, -5185.604004f, 572.490662f, 400.257904f);
+                        AddWaypoint(13, -5189.048828f, 533.551758f, 389.216064f);
+                        AddWaypoint(14, -5188.400391f, 508.496002f, 387.779266f);
+                        AddWaypoint(15, -5182.246582f, 490.885254f, 388.022522f);
+                        AddWaypoint(16, -5167.264160f, 479.733826f, 389.530670f);
+                        AddWaypoint(17, -5157.008301f, 472.717865f, 390.544586f);
+                        AddWaypoint(18, -5144.433594f, 458.945282f, 392.658112f);
+                        AddWaypoint(19, -5129.138184f, 448.644257f, 395.268951f);
+                        AddWaypoint(20, -5113.296875f, 454.575165f, 400.071075f);
+                        AddWaypoint(21, -5106.192871f, 458.708801f, 402.284393f);
+                        AddWaypoint(22, -5100.262695f, 462.922180f, 403.585724f);
+                        AddWaypoint(23, -5093.171387f, 469.209137f, 403.715790f);
+                        break;
+                    default:
+                        sLog->outError("Unexpected movement variation (%i) in npc_og_suitAI::SetupMovement call!", variation);
+                        return;
+                }
+                if (npc_og_suitAI* pEscortAI = CAST_AI(npc_og_suitAI, me->AI()))
+                {
+                    if (variation == 5)
```

Comment:
add in any if/else conditions the curly brackets {} pls

Reply:
fixed

Reply:
@FALL1N1 good point, we usually try to at least include them in newly-added code. But I understand your code is very big and if it takes too much of your time, I can live with it

Reply:
The issue is that it's not fully my code and I haven't reviewed all of it, but I will go through and add brackets wherever necessary.